docs/proposals/20210203-unmanaged-infrastructure.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Externally Managed cluster infastructure
+authors:
+  - "@enxebre"
+  - "@joelspeed"
+  - "@alexander-demichev"
+reviewers:
+  - "@vincepri"
+  - "@randomvariable"
+  - "@CecileRobertMichon"
+  - "@yastij"
+creation-date: 2021-02-03
+last-updated: 2021-02-12
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Externally Managed cluster infrastucture
+
+## Table of Contents
+
+A table of contents is helpful for quickly jumping to sections of a proposal and for highlighting
+any additional information provided beyond the standard proposal template.
+[Tools for generating](https://github.com/ekalinin/github-markdown-toc) a table of contents from markdown are available.
+
+- [Title](#title)
```

Comment:
Do we need to regenerate this TOC before we can merge?

Reply:
not sure how to do it, is there any automation for this?

Reply:
https://ecotrust-canada.github.io/markdown-toc/  or any other software you can find :)

Reply:
Markdown all in one in VS Code will regenerate TOCs for you.